libvirt/tests/src/libvirt_mem.py

```
@@ -116,6 +116,56 @@ def restore_hugepages(page_size=4):
+        Must be called when no VM is using vhost modules.
+        This function applies runtime configuration only.
+        For persistent configuration across reboots, manually create:
+        /etc/modprobe.d/vhost.conf with content: options vhost max_mem_regions=512
+        """
+        try:
+            # Check if running on SLES
+            distro_check = process.run("cat /etc/os-release | grep -i sles",
+                                      shell=True, ignore_status=True)
+            if distro_check.exit_status == 0:
+                logging.info("SLES detected, configuring vhost max_mem_regions=512")
+                vm_check = process.run("virsh list --state-running | grep -v 'Id\\|^$\\|^-'",
```

Ruff (0.15.15)

[error] Function call with `shell=True` parameter identified, security issue

(S604)